Annotato/Annotato/Views/Components/Extensions/UIView.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+extension UIView {
+    func showSelected() {
+        let oldColor = backgroundColor
+        let duration = 0.2
+        let red: CGFloat = 245 / 255
+        let green: CGFloat = 234 / 255
+        let blue: CGFloat = 105 / 255
+        let alpha: CGFloat = 0.3
+        let color = UIColor(red: red, green: green, blue: blue, alpha: alpha)
```

Comment:
```suggestion
        let color =  #colorLiteral(red: 245 / 255, green: 234 / 255, blue: 105 / 255, alpha: 0.3)
```

Is it better to use this instead? Can see the color in xcode.

Reply:
It doesn't seem to work
<img width="537" alt="Screenshot 2022-03-18 at 10 17 01 AM" src="https://user-images.githubusercontent.com/71375383/158924705-5d23ba62-1fa1-4e13-bb12-e53d619e3cb1.png">